docs/getting-started/installing-data.md

```diff
@@ -12,9 +12,14 @@ There are two options for installation:
 
 ## Installing binaries
 
-1. Go to the [releases page](https://github.com/datahq/datahub-cli/releases/latest)
+1. Go to the [releases page](/download)
 2. Download the pre-built binary for your platform (MacOS and LinuxOS x64 at present)
-3. Move the binary into your `$PATH` e.g. on Mac you could move to `/usr/local/bin/`
+3. Make it executable by running 
+
+    ```bash
+    $ chmod +x data-{os-distribution}
```

Comment:
Is this issue with mac-os as well? Ar you sure the command same?

Reply:
Yes, same with mac-os